Gruntfile.js

```diff
@@ -195,37 +202,30 @@ module.exports = function( grunt ) {
 	} );
 
 	grunt.registerTask( "lint", [
-		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint:dev" ),
+		runIfNewNode( "newer:eslint:dist" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
```

Comment:
Because you submitted it only yesterday so few people had a chance to look (I don't think we should land PRs not LGTM'd by at least 2 core members after only a day of submitting a PR) at it **and** the two people that have looked at it have comments.